conf/nginx.conf.server

```diff
@@ -138,6 +138,12 @@ server {
         proxy_cache_use_stale error timeout updating;
         proxy_cache_background_update on;
 
+        # These timeouts ensure that the stale cache is used quickly if network
+        # goes down. The timeouts are for time to first byte, not entire request duration.
+        proxy_connect_timeout 1s;
```

Comment:
@concreted: Is 1s lenient enough? What led you to 1s?

Reply:
This timeout is for whether the upstream server is reachable. The upstream in this case is another Nginx process running on the same box, so it should always be reachable with millisecond latency as long as Nginx is running. The default is 60s, and wanted to have an explicit timeout set for this since we are setting the other two here.

Reply:
👍